### ISSUE TITLE
Newsletter

### ISSUE THREAD

**Issue by gyachdav** [Rostlab/JS16_ProjectF#477]
Please share your thoughts anoint incorporating a newsletter signup into the site? Good/bad/irrelevant?

**Comment by jorjo1** [comment]
It's cool. Now I'd say it's lower priority while we have these social pages -fb,twitter etc.
I did a research about mailing services and the most services are paid or limited in freemium to about 2k users.. so the best solution would be to host it on our lrz server.

**Comment by gyachdav** [comment]
Why low priority? we get mad traffic. I need a time/effort estimate for this feature.

We'll figure out what to do/how to use the list later.

**Comment by jorjo1** [comment]
I agree on this, we get really crazy stats. As an estimate of time and effort to implement a mailing list feature on the site I'd say it should be less than 2 hours. However, it depends on the platform we choose to store our email subscribers later on.
